CHANGELOG.rst

```diff
@@ -2,6 +2,12 @@
 CHANGELOG
 =========
 
+1.7.3
```

Comment:
Instead, could you move these under the previous ("dev") release, and change that header to "1.8.0dev"?